test/lib/jdk/test/lib/process/ProcessTools.java

```diff
@@ -718,12 +789,12 @@ public OutputStream getOutputStream() {
 
         @Override
         public InputStream getInputStream() {
-            return p.getInputStream();
+            return out;
```

Comment:
This is a little bit confusing that the `getInputStream()` returns `out` stream.
Just wanted to double-check if it is intentional and was not needed for `getOutputStream()` instead.

Reply:
Agree, it is confusing, even in standard j.l.Process API . The `InputStream java.lang.Process.getInputStream()`" returns **output** stream of started process.  So for our implementation ProcessImpl the 'out' and 'err' mean output and error streams. However they are returned as InputStreams so users could read them.

Reply:
Right. From the API caller's POV, it is asking for InputStreams that it can use to read the process' stdout or stderr streams.

Reply:
Okay, thanks.
I'm thinking about simple/short comments on this to make it clear this naming mismatch is intentional.

Reply:
There is a comment in line 156 explaining the purpose of these streams.
Also, I renamed out/err names to stdOut/stdErr to make it clearer.